jobs/build/rhcos_sync/build.groovy

```diff
@@ -102,6 +102,10 @@ def rhcosSyncMirrorArtifacts() {
     buildlib.invoke_on_use_mirror("rhcossync.sh", invokeOpts)
 }
 
+def rhcosSyncROSA() {
+    sh("${env.WORKSPACE}/build-scripts/rosa-sync/rosa_sync.sh ${params.BUILD_VERSION} ${rhcosWorking}/meta.json ${params.DRY_RUN}")
```

Comment:
@LorbusChris just wanted to confirm that Jenkins `$BUILD_VERSION` equates to the version looking like this `rhcos-46.82.202008181646-0` ?

Reply:
I think it expands closer to the OCP versioning based on [this snippet](https://github.com/openshift/aos-cd-jobs/blob/7a4df65fc0cf5c527ce59a0f5b04f65bf4a61409/jobs/build/rhcos_sync/Jenkinsfile#L36-L44). However, `$RHCOS_BUILD` [is very close](https://github.com/openshift/aos-cd-jobs/blob/7a4df65fc0cf5c527ce59a0f5b04f65bf4a61409/jobs/build/rhcos_sync/Jenkinsfile#L55-L60). It's the RHCOS version without the prefix of `rhcos-`.

Reply:
it does not :) I've changed it so it pulls the actual name from the original AMI.